.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+<!-- Please tick at least one of the following: -->
+
+- [ ] I have read the [CONTRIBUTING guidelines](README.md#user-content-contributing)
+- [ ] A GitHub issue was raised for this feature / bug or it was discussed on the projects chat
```

Comment:
Could you add the checklist here with some of the items from the issue?

Reply:
Oops, sorry. Overlooked that you actually had an example xD

Adding the checkpoints…